.github/workflows/docker-build-publish.yaml

```diff
@@ -26,7 +26,8 @@ on:
 env:
   DOCKER_BUILDX_PLATFORM: linux/amd64
   DOCKER_REGISTRY_OWNER: athenz
-  GOLANG_VERSION: ^1.19
+  DOCKER_REGISTRY_USER: abvaidya
+  GOLANG_VERSION: ^1.21
```

Comment:
@WindzCUHK Shouldn't we stop defining the specific version and set `go-version: 'stable'`?

Reply:
fixed in: [028ecad](https://github.com/AthenZ/authorization-proxy/pull/26/commits/028ecad674f15835b88d407de220d3cb34a77fd4)